src/index.js

```diff
@@ -25,17 +25,22 @@ class RetryChunkLoadPlugin {
           }
         `;
 
-          const cacheBust = this.options.cacheBust
-            ? `
-          (${this.options.cacheBust})();
-        `
-            : '"cache-bust=true"';
+          const getCacheBustString = () =>
+            this.options.cacheBust
+              ? `
+            (${this.options.cacheBust})();
+          `
+              : '"cache-bust=true"';
```

Comment:
I think default one should be changed to include retry attempt number (or something else dynamic) so retry attempts aren't cached.

Reply:
True.
Have updated the code to append retryAttempt along with the custom cache-string.

Reply:
Have also updated the e2e test to verify this behaviour as well.